gtsfm/utils/geometry_comparisons.py

```diff
@@ -40,6 +43,115 @@ def align_rotations(aRi_list: List[Rot3], bRi_list: List[Rot3]) -> List[Rot3]:
     return [aRb.compose(bRi) for bRi in bRi_list]
 
 
+def ransac_align_poses_sim3_ignore_missing(
```

Comment:
Thanks for the review Ayush. I think that making a RANSAC wrapper for this will make the code a bit less readable. Since we'll have to have a very generic interface. So I'd prefer we leave it as is, for now.